Added Libxml2 and Macs2 files (#311)
odidev/ecosystem-dashboard-for-arm: content/opensource_packages/macs2.md

---
name: MACS2
category: HPC
description: Model-based Analysis of ChIP-Seq (MACS2) is a bioinformatics tool used for identifying enriched regions in ChIP-Seq data, determining where DNA-binding proteins interact with the genome.
download_url: https://github.com/macs3-project/MACS/releases
works_on_arm: true
supported_minimum_version:
    version_number: 2.0.10.07022012
    release_date: 2012/07/03
 
 
optional_info:
    homepage_url: https://macs3-project.github.io/MACS/
    support_caveats:
    alternative_options:
    getting_started_resources:
        arm_content: 
        partner_content:
        official_docs: https://macs3-project.github.io/MACS/docs/INSTALL.html
    arm_recommended_minimum_version:
        version_number:
        release_date:
 
optional_hidden_info:
    release_notes__supported_minimum:
    release_notes__recommended_minimum:

>     other_info: Successfully installed minimum version available(2.0.10.07022012) in relese page on Arm64 platform.

Author: ```suggestion
    other_info: Successfully installed minimum version available(2.0.10.07022012) from release page for Arm64 platform.
```
